module2/projects/peer_code_share.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: page
+title: Relational Rails - Peer Code Share
+---
+
+_[Back to Relational Rails Home](./relational_rails)_
+
+## Peer Code Share
+
+You will be partnered with another student to review each other's code and provide feedback.
+
+First, exchange Github repository links with the other student, and clone the repository. Set up the application locally by first checking that *their* database is named differently than *yours* - if you `drop` or re-seed a database of the exact same name as your project's, then yours will be affected too! To accomplish this, you could simply add the name of the student at the start of the database name, for example: `brian_relational_rails_development`, `brian_relational_rails_test`. You can find the database configuration in `config/database.yml`. 
```

Comment:
I think we can make this part optional. It should be okay if a students drops their development database for their project to view their partner's project. I can see some students getting hung up on changing the database names.

Reply:
Agreed - good point. I'll remove this part for now but leave the instructions for what to run in the terminal (which incidentally is the same steps for them to remake their database if needed anyway).